doc/api/util.md

```diff
@@ -620,8 +620,35 @@ util.inspect(obj);
 added: v6.6.0
 -->
 
-* {symbol} that can be used to declare custom inspect functions, see
-[Custom inspection functions on Objects][].
+* {symbol} that can be used to declare custom inspect functions.
+
+This is a shared symbol, `Symbol.for('util.inspect.custom')`, which is
+registered in the [global symbol registry][]. It can be used to define
+custom inspection functions in any environment.
+
+```js
+const inspect = Symbol.for('util.inspect.custom');
+
+class Password {
+  constructor(value) {
+    this.value = value || document.getElementById('password').value;
+  }
+
+  toString() {
+    return 'xxxxxxxx';
+  }
+
+  [inspect]() {
+    return this.toString();
+  }
+}
+
+const password = new Password('r0sebud');
+console.log(password);
+// Prints xxxxxxxx
```

Comment:
It might be obvious from context, but when only considering the example, it’s not all that obvious whether this prints `xxxxxxxx` because of `.toString()` or because of the `[inspect]()` method.

Maybe we could change the latter one to something like ``return `Password <${this.toString()}>`; ``?

Reply:
[Done](https://github.com/nodejs/node/pull/20857/commits/11a48cd4bb9bbe38e3063c78594fb98915718c00).